roxctl/helm/derivelocalvalues/k8s_access.go

```diff
@@ -117,6 +117,17 @@ func (k *k8sObjectDescription) evaluateToString(ctx context.Context, kind string
 	return s
 }
 
+func (k *k8sObjectDescription) evaluateToStringSlice(ctx context.Context, kind string, name string,
+	jsonpath string, def []string) []string {
+	x := k.evaluateOrDefault(ctx, kind, name, jsonpath, def)
+	s, ok := x.([]string)
+	if !ok {
+		k.warn("Unexpected data type (%T) at JsonPath %q for resource %s/%s", x, jsonpath, kind, name)
```

Comment:
Perhaps for me as a user it would be also beneficial to learn which value was used instead

Reply:
Hm, yeah we only print the data type that was used. I've added to also print the value, and adjusted all other messages that print the info.